collections/internal/coretesting/context.go

```diff
@@ -0,0 +1,32 @@
+package coretesting
+
+import (
+	"context"
+
+	"cosmossdk.io/collections/corecompat"
+)
+
+type dummyKey struct{}
+
+func Context() context.Context {
```

Comment:
I think instead of defining this stuff we should just go back to how we tested and created a test db before:

https://github.com/cosmos/cosmos-sdk/blob/0e8cef5795cefadcea83a0b4815c843dc70ecaa9/collections/collections_test.go#L48

This is a ref from collections/v0.4.0

Reply:
How's this: https://github.com/cosmos/cosmos-sdk/pull/24081/commits/1efb5b2709b95d0ea9770d6a6c3f85f53227c6df ?